tree/dataframe/src/RDFBookedDefines.cxx

```diff
@@ -13,8 +13,10 @@ bool RBookedDefines::HasName(std::string_view name) const
 void RBookedDefines::AddColumn(const std::shared_ptr<RDFDetail::RDefineBase> &column, std::string_view name)
 {
    auto newCols = std::make_shared<RDefineBasePtrMap_t>(GetColumns());
-   (*newCols)[std::string(name)] = column;
+   const auto colName = std::string(name);
```

Comment:
And that's better than
```suggestion
   const std::string colName(name);
```
?

Reply:
if you ask Herb Sutter, yes :smile: works with every type and you never incur in most vexing parse issues.

(I don't really have an opinion, I don't know why I wrote it like that)